### PR TITLE
Updated teller classes into banking package

### PR DESCRIPTION
**Pull request by hle95**
Added new Constructor for Employee with employeeType
Updated Teller from default package to new package
Added TellerGUI
Added TellerDriver
Updatee EmployeeFileHandler to store and write employee
NO ONLINE/SHUTDOWN at the moment

### CODE REVIEW

File: Banking/src/banking_dev/Teller.java [hunk 1]
```diff
@@ -198,4 +196,141 @@ public void listen() throws IOException, ClassNotFoundException {
...
+	public String getBalance(int accountID) {
+		Money balance = customer.findAccount(accountID).getBalance();
+
+		return "" + balance;
+	}
+
+	public Boolean addEmployee(String name, String username, String password, String employeeType)
+			throws IOException, ClassNotFoundException {
+		if (employeeType.toUpperCase() == "EMPLOYEE") {
+			Employee newEmployee = new Employee(name, username, password, EmployeeType.EMPLOYEE);
```
Comment (AEGarten) [inline Banking/src/banking_dev/Teller.java]: Not an Employee object. It should be a NewEmployee message. I need to create this. It's constructor will be NewEmployee(Employee) or something.

**Review by AEGarten — APPROVED** [review]
New Employee needs to pass a NewEmployee message not just an Employee. Also make a new package for  testdrivers  (named exactly just like that) and put all your test drivers in there so GitHub won't track those.

File: Banking/src/banking_dev/Teller.java (hunk 1 above)
Comment (hle95) [reply Banking/src/banking_dev/Teller.java]: Updated

**Review by AEGarten — APPROVED** [review]
great

**Pull request merged**